WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			independent = yes
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
 			}
 			is_adult = yes
-			prestige = 5000
+			prestige = 2000
```

Comment:
I cannot set out to convince you, because you are the most stubborn, hard-headed man I've had to work with in years. Nothing phases your iron will or your greed in demanding everything be exactly perfect in your mind, regardless of my justifications.